### PR TITLE
Publish 1.3.2 (published to tag)

### PR DESCRIPTION
**Pull request by Manishearth**
This brings the commits from https://github.com/unicode-org/icu4x/pull/4107 onto main. ICU4X 1.3.2 was *not* published from this commit, it was published from ac39015, which has been tagged as `icu@1.3.2`. This simply brings main into sync

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -91,7 +91,6 @@ members = [
     "experimental/zerotrie",
 
     # Tools
-    "tools/bakeddata-scripts",
```
Comment (robertbastian) [inline Cargo.toml:94]: no

**Comment by Manishearth** [comment]
patch sgtm, can someone rereview with autosubmit?

**Comment by sffc** [comment]
I didn't catch why we're on 1.3.2 and not 1.3.1?

**Comment by Manishearth** [comment]
@sffc some crates already had a 1.3.1 and this is a combined release so we picked the lowest fresh version to give everyone

**Pull request merged**